ProjectHeads/App.Head.WinAppSdk.props

```diff
@@ -3,6 +3,10 @@
     <DependencyVariant>WinUI</DependencyVariant>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+      <PublishAot>true</PublishAot>
```

Comment:
This is incorrect. It should not be behind a condition, but rather always set for all configurations, unconditionally.

Reply:
> "Are there docs on this?"

Yes, but they are wrong 😆
Being fixed in WinAppSDK 1.6-preview1.

> "I'm not clear on why it's different for wasdk"

This has nothing to do with WinAppSDK, it's just a .NET thing. `PublishAot` does **not** mean "run on NativeAOT. It actually means: "run in AOT-emulation mode on CoreCLR for normal builds, and **publish** with NativeAOT when you run the publish targets for your project". So the point is to always keep it enabled, so that even when you're doing normal Debug builds, you still get all warnings and all behaviors you'll get in AOT (eg. code paths that check for AOT will think they're running on AOT even though they're not). This gives you a good debugging experience (as you can use the managed .NET debugger), and ensures that your app behaves the same on NAOT as well, which avoids things randomly breaking only when publishing (which is what happens all the time with .NET Native instead, which isn't great).

Reply:
That makes sense, thanks for clearing it up @Sergio0694!